File: backend/api/Models/Entry.cs
namespace xmas.Models;

public class Entry 
{
    public int Id { get; set; }

    public int Day { get; set; }
    public DateOnly Date { get; set; }

    public string? Title { get; set; }
    public string? InnerHTML { get; set; }

    public List<byte>? Image { get; set; } 
    public List<byte>? Thumbnail { get; set; }
}

File: front/dev/ts/entry.ts
export class FullEntryDto
{
    day: number;
    date: number;

    title: string;
    innerHTML: string;

    image: string;
    thumbnail: string;
}

export class FullEntry
{
    day: number;
    date: Date;

    title: string;
    innerHTML: string;

    image: Blob;
    thumbnail: Blob;
}

export class ThumbEntryDto
{
    day: number;
    date: number;

    title: string;

    thumbnail:string;
}

export class ThumbEntry
{
    day: number;
    date: Date;

    title: string;

    thumbnail: Blob;
}
